notebooks/api/0.8/06-domain-register-control-flow.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
currently all of the api/0.8 notebooks contain more or less full user stories, while the tutorials are more about "how do I do X". I think toggling this setting would be a better fit in the tutorials. If there is anything missing there we can move it there. Would you agree?

Reply:
Yes, I agree. Currently we have added it both places. 0.8/ has a notebook with asserts and then we have added the same to the account settings tutorial (scroll to the end).
https://github.com/OpenMined/PySyft/blob/98f8eae95223c650ed12ae6b486c7d2fcb143376/notebooks/tutorials/data-owner/02-account-management.ipynb
Do you think we need a separate notebook instead ?

Reply:
I meant to propose to remove the notebook from 0.8/api, as this is just a minor setting and not really a full user flow

Reply:
but if you feel like it should stay here thats also fine with me